ios/browser/api/password/password_store_listener_ios.mm

```diff
@@ -0,0 +1,137 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/ios/browser/api/password/password_store_listener_ios.h"
+
+#include "base/check.h"
+#include "base/strings/sys_string_conversions.h"
+
+#include "components/password_manager/core/browser/password_form.h"
+#include "components/password_manager/core/browser/password_store_interface.h"
+
+#include "net/base/mac/url_conversions.h"
+
+#include "brave/ios/browser/api/password/brave_password_api.h"
+#include "brave/ios/browser/api/password/brave_password_observer.h"
+
+#if !defined(__has_feature) || !__has_feature(objc_arc)
+#error "This file requires ARC support."
+#endif
+
+@interface IOSPasswordForm (Private)
+
+- (instancetype)initWithURL:(NSURL*)url
+                signOnRealm:(NSString*)signOnRealm
+                dateCreated:(NSDate*)dateCreated
+               dateLastUsed:(NSDate*)dateLastUsed
+        datePasswordChanged:(NSDate*)datePasswordChanged
+            usernameElement:(NSString*)usernameElement
+              usernameValue:(NSString*)usernameValue
+            passwordElement:(NSString*)passwordElement
+              passwordValue:(NSString*)passwordValue
+            isBlockedByUser:(bool)isBlockedByUser
+                     scheme:(PasswordFormScheme)scheme;
```

Comment:
Isn't this the same init signature as in `brave_password_api.h`?

https://github.com/brave/brave-core/pull/12360/files#diff-e843deb51b88d9fb7e5670a417df620962db0f68fb0664fc0323f2e6b14cd225R60-R70

Reply:
Correct, there is no need for this since the header is imported. 
Removed